moveit_ros/planning/planning_pipeline/include/moveit/planning_pipeline/planning_pipeline.h

```diff
@@ -124,8 +125,7 @@ class PlanningPipeline
       \param req The request for motion planning
       \param res The motion planning response */
   bool generatePlan(const planning_scene::PlanningSceneConstPtr& planning_scene,
-                    const planning_interface::MotionPlanRequest& req,
-                    planning_interface::MotionPlanResponse& res) const;
+                    const planning_interface::MotionPlanRequest& req, planning_interface::MotionPlanResponse& res);
```

Comment:
I guess removing constness for both generatePlan() methods was "required" to modify the `active_` flag.
Instead of giving up constness here, I suggest changing the `active_` member to `mutable`.

Reply:
Good idea!